secator/runners/_helpers.py

```
@@ -6,6 +6,42 @@
+	to_remove = []
+	resolved = {}
+	for key, val in opts.items():
+		if not key.endswith('_'):
+			continue
+		if not isinstance(val, dict) or 'condition' not in val:
+			continue
+		base_key = key.rstrip('_')
+		condition = val['condition']
+		opt_value = val.get('value', True)
+		try:
+			if eval(condition, safe_globals, context):
```

Ruff (0.15.10)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)